### DISCUSSION TITLE
MFA lost

### DISCUSSION THREAD

**Discussion by silverraindog** [community/community#37889] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi

So one of the senior devs left the company 2 or 3 years ago. I don't think we were using github even then. I am now trying to login to github with his details but getting hit by the MFA. I have done the "password reset" and  get to the "recovering your account". Which all is greyed out. Assuming he did not have personal device token, ssh or anything. 

How would we recover his account ? Since we are not using github and being charged for it, we would like to cancel. The email address in question is ian [at] bettech [dot] com or software at the same domain

**Comment by airtower-luna** [comment, marked as answer]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

**Reply by silverraindog** [reply]
@airtower-luna Thank you, I have messaged them.
